src/ai_hawk/llm/llm_manager.py

```diff
@@ -584,9 +590,10 @@ def answer_question_textual_wide_range(self, question: str) -> str:
         if chain is None:
             logger.error(f"Chain not defined for section '{section_name}'")
             raise ValueError(f"Chain not defined for section '{section_name}'")
-        output = chain.invoke(
+        raw_output = chain.invoke(
             {RESUME_SECTION: resume_section, QUESTION: question}
         )
+        output = self._clean_llm_output(raw_output)
```

Comment:
I believe sometimes these questions not only be one word answer questions.

something like "why are  you interested in <compnay name>?"

* serve as bullet points